src/chloadaddedmass.cpp

```diff
@@ -13,14 +13,18 @@
  * initializes body to have load applied to and added mass matrix from h5 file object
  *******************************************************************************/
 ChLoadAddedMass::ChLoadAddedMass(const std::vector<H5FileInfo>& user_h5_body_data,
-                                 std::vector<std::shared_ptr<ChLoadable>>& bodies)
-    : ChLoadCustomMultiple(bodies) {
-	nBodies = bodies.size();
+	std::vector<std::shared_ptr<ChLoadable>>& bodies,
+	ChSystem* system)
+	: ChLoadCustomMultiple(bodies), system(system) {
+	auto nBodies = bodies.size();
 
 	infinite_added_mass.setZero(6 * nBodies, 6 * nBodies);
 	for (int i = 0; i < nBodies; i++) {
 		infinite_added_mass.block(i * 6, 0, 6, nBodies * 6) = user_h5_body_data[i].GetInfAddedMassMatrix();
 	}
+
+	// initialize added mass matrix for whole system
+	infinite_added_mass_system = infinite_added_mass;
```

Comment:
Does it make sense to move lines 41-55 below to the constructor here? I think it's fine for now either way, but just wondering if we want to move that set up step to the constructor so it only happens once a simulation. Later if we decide to write a separate function to handle the infinite_added_mass_system variable set up, I think we would still call it from this line?